src/general_manager/cache/cacheDecorator.py

```
@@ -1,70 +1,46 @@
-from typing import Callable, Optional
+from typing import Callable, Optional, Protocol, Any, Optional, Set, Callable
```

Ruff (0.11.9)

Redefinition of unused `Optional` from line 1

Remove definition: `Optional`

(F811)